response/slack/decorators/action_handler.py

```diff
@@ -11,13 +11,11 @@
 
```

Comment:
I've added other action types to block_kit so needed to generalise how we handle action values.